lib/core-integration/src/Test/Integration/Scenario/API/Shelley/Transactions.hs

```diff
@@ -2818,11 +2920,47 @@ spec = describe "SHELLEY_TRANSACTIONS" $ do
                 "passphrase": #{passphrase}
             }|]
 
+    pickAnAsset :: TokenMap.TokenMap -> ((Text, Text), Natural)
+    pickAnAsset tm = case TokenMap.toFlatList tm of
+        (TokenBundle.AssetId pid an, TokenBundle.TokenQuantity q):_ ->
+            ((toText pid, toText an), q)
+        _ -> error "pickAnAsset: empty TokenMap"
+
+    -- Like mkTxPayload, except that assets are included in the payment.
+    -- Asset amounts are specified by ((PolicyId Hex, AssetName Hex), amount).
+    mkTxPayloadMA
+        :: MonadUnliftIO m
+        => Context
+        -> ApiWallet
+        -> Natural
+        -> [((Text, Text), Natural)]
```

Comment:
Perhaps it would be nice to add a field comment here or type alias here, just to state what these `Text` values should be.)

Reply:
There is a comment which says `(PolicyId Hex, AssetName Hex)` just above - is that alriight?